src/meta-srv/src/bootstrap.rs

```diff
@@ -278,6 +276,7 @@ pub async fn metasrv_builder(
             let candidate_lease_ttl = Duration::from_secs(CANDIDATE_LEASE_SECS);
             let execution_timeout = Duration::from_secs(META_LEASE_SECS);
             let statement_timeout = Duration::from_secs(META_LEASE_SECS);
+            let idle_session_timeout = Duration::from_secs(META_LEASE_SECS + 1);
```

Comment:
It seems `idle_session_timeout` need to be `<=` `META_LEASE_SECS`

Reply:
resolved in [23f33a4](https://github.com/GreptimeTeam/greptimedb/pull/6341/commits/23f33a494f1bff5d1653a4405ba9bee8eafed744)